src/main/java/apoc/uuid/UuidHandler.java

```diff
@@ -0,0 +1,183 @@
+package apoc.uuid;
+
+import apoc.util.Util;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Label;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.graphdb.event.PropertyEntry;
+import org.neo4j.graphdb.event.TransactionData;
+import org.neo4j.graphdb.event.TransactionEventHandler;
+import org.neo4j.graphdb.schema.ConstraintDefinition;
+import org.neo4j.graphdb.schema.Schema;
+import org.neo4j.kernel.impl.core.EmbeddedProxySPI;
+import org.neo4j.kernel.impl.core.GraphProperties;
+import org.neo4j.kernel.internal.GraphDatabaseAPI;
+import org.neo4j.logging.Log;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+import static apoc.util.Util.map;
+import static apoc.util.Util.toBoolean;
+
+public class UuidHandler implements TransactionEventHandler {
+
+    private static final String APOC_UUID = "apoc.uuid";
+    private static final String UUID_PROPERTY = "uuid";
```

Comment:
shouldn't the property be configurable? Perhaps in follow up-PR?

Reply:
Added in this PR